testing/examples/run_instruction_apt_pkgs/cloudbuild.yaml

```diff
@@ -0,0 +1,35 @@
+# Copyright 2017 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Tests that verify the example installing apt packages using rules_docker
+# and Dockerfile.
+
+timeout: 21600s
+options:
+    machineType: "N1_HIGHCPU_32"
+
+steps:
+  # Confirm bazel and docker are installed.
+  - name: "gcr.io/asci-toolchain/nosla-ubuntu16_04-bazel-docker-gcloud"
+    entrypoint: "sh"
+    args: ["-c", "bazel version && docker version"]
+    id: "versions"
+    waitFor: ["-"]
+
+  # Test the run_instruction_apt_pkgs example.
+  - name: "gcr.io/asci-toolchain/nosla-ubuntu16_04-bazel-docker-gcloud"
+    entrypoint: "sh"
+    args: ["-c", "cd ../testing/examples && bazel --output_base=/workspace/apt_pkgs test //run_instruction_apt_pkgs:all"]
```

Comment:
Why do you need to set the output base?

Reply:
Not a 100% sure, but this makes sure that Bazel generated files can be a accessed by multiple containers (even within the same steps). This is needed here and in the run_instruction_arbitrary example because these examples use the `download_pkgs`, `install_pkgs` and `container_run_and_commit` rules that launch additional containers.

Reply:
So do you get an error if you dont set this?
What type of error?
If its because these rules need to launch additional containers, there should be some other way to avoid issues; e.g., by passing `/var/run/docker.sock:/var/run/docker.sock` in `volumes` (https://cloud.google.com/cloud-build/docs/build-config)

Reply:
Ass discussed separately, added a comment why this is needed.